sdk/eventhub/Azure.ResourceManager.EventHubs/api/Azure.ResourceManager.EventHubs.netstandard2.0.cs

```diff
@@ -794,9 +815,9 @@ public partial class EventHubsNetworkSecurityPerimeter
         public Azure.Core.AzureLocation? Location { get { throw null; } }
         public string PerimeterGuid { get { throw null; } }
     }
-    public partial class EventHubsNetworkSecurityPerimeterConfiguration : Azure.ResourceManager.Models.TrackedResourceData
+    public partial class EventHubsNetworkSecurityPerimeterConfiguration : Azure.ResourceManager.Models.ResourceData
```

Comment:
This is a breaking change, why move the `location` out? What's the impact if move the `location` back as required property?

Reply:
Sorry for inconvenience. It was swagger problem. Now I have fixed the issue.